maven-plugin/src/main/java/org/openmrs/maven/plugins/model/DistroProperties.java

```diff
@@ -191,6 +194,42 @@ public String getName(){
         return getParam("name");
     }
 
+    public String getPropertyPromt(String propertyName){
+        return getParam(String.format(PROPERTY_PROMPT_KEY, propertyName));
+    }
+
+    public String getPropertyDefault(String propertyName){
+        return getParam(String.format(PROPERTY_DEFAULT_VALUE_KEY, propertyName));
+    }
+
+    public String getPropertyValue(String propertyName){
+        return getParam(String.format(PROPERTY_KEY, propertyName));
+    }
+
+    public Set<String> getPropertiesNames(){
+        Set<String> propertiesNames = new HashSet<>();
+        for(Object key: getAllKeys()){
+            if(key.toString().startsWith("property.")){
+                propertiesNames.add(extractPropertyName(key.toString()));
+            }
+        }
+        return propertiesNames;
+    }
+
+    private String extractPropertyName(String key) {
+        int beginIndex = key.indexOf(".");
+        int endIndex = key.indexOf(".prompt");
+        if(endIndex != -1){
+            return key.substring(beginIndex + 1, endIndex);
+        } else  {
+            endIndex = key.indexOf(".default");
```

Comment:
I think StringUtils.stripStart is not the best solution for this. If you try to do StringUtils.stripStart("property.pih.config", "property.") as a result you will get ih.config. 

Secondly I don't think the order is important here ? We just want all the property names, no matter if it is prompt or default